src/test/java/com/kodilla/ecommercee/domain/CartTestSuite.java

```diff
@@ -0,0 +1,77 @@
+package com.kodilla.ecommercee.domain;
+
+import com.kodilla.ecommercee.repository.CartRepository;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+import java.util.List;
+import java.util.Optional;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class CartTestSuite {
+
+    @Autowired
+    CartRepository cartRepository;
+
+    private Cart createTestCart() {
+        return new Cart();
+    }
+
+    @Test
+    public void testSaveCart() {
```

Comment:
Nie uzależniamy testów od twojego stanu w bazie dancyh. 
Jeżeli ktoś by ten test wykonał a by miał już jakieś rekordy w bazie danych to test by nie przeszedł a funkcjonalnosć dobrze by działała.
Proponuję poprostu zliczyc rekordy przed dodaniem i z licyzc rekordy po dodaniu i odjać od siebie.